cmd/commands/tx.go

```diff
@@ -12,7 +12,7 @@ import (
 
 	cmn "github.com/tendermint/go-common"
 	client "github.com/tendermint/go-rpc/client"
-	"github.com/tendermint/go-wire"
+	wire "github.com/tendermint/go-wire"
```

Comment:
Holy smoke! I thought it just works that way. Just found https://github.com/fatih/vim-go/issues/1237, which looks similar (except I am not linking any gopath).